types/office-js-preview/index.d.ts

```diff
@@ -1443,6 +1457,51 @@ declare namespace Office {
          */
         openBrowserWindow(url: string): void;
     }
+    /**
+     * Provides the runtime URL of an add-in.
```

Comment:
Relatedly, if the plan is to make this API ready for future upgrades, consider leaving the class-level description general enough to accommodate that.

Reply:
Fixed.